staging/src/k8s.io/apimachinery/pkg/util/proxy/dial.go

```diff
@@ -24,10 +24,9 @@ import (
 	"net/http"
```

Comment:
verified this diff results in identical content to release-1.24 branch for this file